sheetScript.gs

```diff
@@ -173,6 +244,18 @@ function getVideoIds(channelId, lastTimestamp) {
 
 function getPlaylistVideoIds(playlistId, lastTimestamp) {
   var videoIds = [];
+  
+  // Check Playlist validity
+  var results = YouTube.Playlists.list('id', {
+    id: playlistId
+  });
+  if (!results || !results.items) {
+    Logger.log("YouTube channel search returned invalid response for playlist with id "+playlistId)
+    return []
+  } else if (results.items.length === 0) {
+    Logger.log("Cannot find playlist with id "+playlistId)
+    return []
+  }
```

Comment:
Does this double the quota?

Reply:
Moved so that it only checks if no videos are found